tsconfig.json

```diff
@@ -10,6 +10,7 @@
 		"strict": true,
 		"forceConsistentCasingInFileNames": true,
 		"esModuleInterop": true,
-		"baseUrl": "."
-	}
+		"baseUrl": "./",
+	},
+//	"include": ["src"]
```

Comment:
Let's revert these changes, since it looks like its just commented code and the baseUrl is effectively the same